### ISSUE TITLE
Flutter Access

### ISSUE THREAD

**Issue by Dokotela** [GoogleCloudPlatform/healthcare#480]
I've been building a FHIR library for Dart/Flutter (https://github.com/fhir-fli/fhir). All resources are defined, it will securely store data on the device, and it's able to make all of the RESTful calls according to the FHIR spec. But I've run into an issue with authentication and authorization. Should I be able to access a fhirstore through an android app? Also, have you had any thought to modeling a FHIR database in cloud firestore?

**Comment by yuryninog** [comment]
Hi! @Dokotela could you confirm please if you moved this repository? please ... I was trying to access https://github.com/fhir-fli/fhir but it does not work.

**Comment by Dokotela** [comment]
Of course! The repo has actually been moved here: https://github.com/MayJuun/fhir, but checkout the dev branch for the most recent developments.
